lib/todolist/tasks/task.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Todolist.Tasks.Task do
+  use Ecto.Schema
+  use Pow.Ecto.Schema
+  alias Todolist.Repo
+
+  schema "task" do
+    field(:name, :string)
+
+    timestamps()
+  end
+
+  def get_all() do
```

Comment:
This should be in the context module for this resource (eg. `lib/todolist/tasks.ex`)